lib/compress.js

```diff
@@ -2757,12 +2761,16 @@ merge(Compressor.prototype, {
     });
 
     OPT(AST_Conditional, function(self, compressor){
-        if (!compressor.option("conditionals")) return self;
-        if (self.condition instanceof AST_Seq) {
-            var car = self.condition.car;
-            self.condition = self.condition.cdr;
-            return AST_Seq.cons(car, self);
```

Comment:
This is basically `lift_sequences()` in disguise, so I think putting it under `options.sequences` as with the case of `AST_Unary` and `AST_Binary` would make it more consistent.